src/activemasternode.cpp

```diff
@@ -127,6 +130,14 @@ bool CActiveMasternode::SendMasternodePing()
     return true;
 }
 
+bool CActiveMasternode::UpdateSentinelPing(int version)
```

Comment:
I've thought that VersionInfo is convertor,  not storage for version information. But if you think it is better to use it to store the version itself,  I change this variables type.